### ISSUE TITLE
[Accessibility] Name on card autocomplete

### ISSUE THREAD

**Issue by gazjoy** [Adyen/adyen-web#1690]
**Describe the bug**
The Adyen drop-in component "name on card" field does not have an autocomplete attribute identifying the input purpose. [WCAG 2.1 - 1.3.5 Identify Input Purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html) describes the guidance. This input field is serving a purpose of cc-name from the [list of input purposes](https://www.w3.org/TR/WCAG21/#input-purposes).

**To Reproduce**
1. View HTML for field - input name="holderName" 
2. See no autocomplete attribute

**Expected behavior**
The field to have the autocomplete with cc-name as the value <input name="holderName" autocomplete="cc-name" />

**Screenshots**
<img width="696" alt="image" src="https://user-images.githubusercontent.com/628064/184645495-3653e2a2-7f83-4158-84bf-732ce5b682ed.png">

**Desktop (please complete the following information):**
 - OS: macOS Monterey 12.5
 - Browser: Safari 15.6

**Additional context**
SC 1.3.5 Input purpose https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html
WCAG list of input purposes: https://www.w3.org/TR/WCAG21/#input-purposes

**Comment by sponglord** [comment]
Hi @gazjoy - thanks for pointing this out

**Comment by ribeiroguilherme** [comment]
Hi @gazjoy , we released the version 5.23.0 which contains the fix for this problem. Thanks for reaching out to us!
